### DISCUSSION TITLE
Question Raspberry PI

### DISCUSSION THREAD

**Discussion by Juan1ll0** [halfgaar/FlashMQ#13] [General]
Is possible to compile in a Raspberry PI?. Or is a package for Raspbian?

**Comment by halfgaar** [comment]
It compiles with an ARM compiler just fine, but you won't have SSE4.2 optimizations. You can just install `build-essential` and `cmake` and the build script should work.

**Comment by redthor** [comment]
Hi there - I commented out this line in CMakeLists.txt:
```
#SET(CMAKE_CXX_FLAGS "-msse4.2")
```

But then I get this error:
```
In file included from /root/FlashMQ/settings.h:27,
                 from /root/FlashMQ/configfileparser.h:31,
                 from /root/FlashMQ/threaddata.h:37,
                 from /root/FlashMQ/mainapp.h:36,
                 from /root/FlashMQ/mainapp.cpp:18:
/root/FlashMQ/network.h:24:10: fatal error: emmintrin.h: No such file or directory
 #include <emmintrin.h>
          ^~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/flashmq.dir/build.make:63: CMakeFiles/flashmq.dir/mainapp.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:76: CMakeFiles/flashmq.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
```
any idea what to do here?

**Reply by sikmir** [reply]
I have the same problem.

**Reply by halfgaar** [reply]
I missed your message somehow, five days ago.

I just pushed a branch named `remove-sse-from-network-matcher`. Can you check that out and try it?

**Reply by sikmir** [reply]
It builds now!

**Comment by halfgaar** [comment]
Solved.
